rdf-star-cg-spec.html

```diff
@@ -558,22 +562,119 @@ <h2>Evaluation Semantics</h2>
+        </p>
+        <dl>
+            <dt>An <a>embedded</a> triple with subject RDF term <em>S</em>, predicate RDF term <em>P</em> and object RDF term <em>O</em></dt>
+            <dd>
+              <pre>
+                {
+                  "type": "triple",
+                  "value": {
+                     "subject": <i><b>S</b></i>,
+                     "predicate": <i><b>P</b></i>,
+                     "object": <i><b>O</b></i>
+                  }
+                }
+              </pre>
+            </dd>
+          </dl>
+        <div class="example">
+            Consider the following RDF term, an <a>embedded</a> triple in Turtle* syntax:
+          <pre data-transform="updateExample"
+            data-content-type="text/x-turtle-star"
+            class="nohighlight example"
+          >
+            <!--
+            << <http://example.org/alice> <http://example.org/name> "Alice" >>
+            -->
+          </pre>
+          This term is represented in JSON as follows:
+          <pre>
+                {
+                  "type": "triple",
+                  "value": {
+                     "subject": {
+                        "type": "uri",
+                        "value" "http://example.org/alice"
+                     },
+                     "predicate": {
+                        "type": "uri",
+                        "value" "http://example.org/name"
+                     },
+                     "object": {
+                        "type": "literal",
+                        "value" "Alice",
+                        "datatype": "http://www.w3.org/2001/XMLSchema#string"
+                     },
+                  }
+                }
+          </pre>
+        </div>
+      <!-- <div class="issue" data-number="13"></div> -->
       </section>
 
       <section>
         <h2>SPARQL* Query Results XML Format</h2>
+       <p>
+        The result of a SPARQL SELECT query is serialized in XML as defined in [[[rdf-sparql-XMLres]]]. This format proposes an XML representation of variable bindings to RDF terms.
+       </p>
+       <p>To accomodate the new RDF term for <a>embedded</a> triples that RDF* introduces, the list of RDF terms and their XML representations in [<a href="rdf-sparql-XMLres#results">rdf-sparql-XMLres, Section 2.3.1</a>] is extended as follows:
+        </p>
+        <p>
+        <dl>
+            <dt>An <a>embedded</a> triple with subject term <em>S</em>, predicate term <em>P</em>, and object term <em>O</em></dt>
```

Comment:
```suggestion
            <dt>An <a>embedded</a> triple with subject term `S`, predicate term `P`, and object term `O`</dt>
```
Also, same remark as above regarding `table` vs. `dl`.